cmd/minishift/cmd/openshift/config.go

```diff
@@ -27,8 +27,8 @@ const (
 
 var configCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
-	Short: "Displays or patches OpenShift configuration.",
-	Long:  "Displays or patches OpenShift master or node configuration. Patches are supplied in JSON format.",
+	Short: "Displays or updates the OpenShift configuration.",
```

Comment:
Ok I will just use "patching" because tbh it's not that critical and I already spent too much energy trying to work around it. Will update the PR after I fix the integration test and other feedback.